TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpHandler.java

```diff
@@ -60,46 +89,50 @@ public HttpMessage createMessage(String[] initialLine) throws Exception {
                     , new PassThruHttpHeaders(headersToPreserve)
             );
         }
-    }
 
+        @Override
+        public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception {
+            if (captureState.captureIgnoreState == CaptureIgnoreState.IGNORE_RESPONSE) {
+                captureState.captureIgnoreState = CaptureIgnoreState.CAPTURE;
+            }
```

Comment:
Trying to understand this... If the state is to ignore response and we have a read we will set to capture. Is that because reads in this case will only be for request packets and we can reset back to capture or is something else happening here?

Reply:
The default assumption is that all requests and their responses will be captured until a request is marked to IGNORE_REQUEST.  This is a reset since a new request has begun... and yes, reads are only for requests and writes are only for responses and that's why we now have two states for ignore.